bin/wiki_entity_linking/wikidata_pretrain_kb.py

```diff
@@ -13,63 +13,73 @@
 """
 from __future__ import unicode_literals
 
-import datetime
+import logging
 from pathlib import Path
 import plac
 
-from bin.wiki_entity_linking import wikipedia_processor as wp
+from bin.wiki_entity_linking import wikipedia_processor as wp, wikidata_processor as wd
 from bin.wiki_entity_linking import kb_creator
-
+from bin.wiki_entity_linking import training_set_creator
+from bin.wiki_entity_linking import TRAINING_DATA_FILE, KB_FILE, KB_MODEL_DIR
 import spacy
 
-from spacy import Errors
-
-
-def now():
-    return datetime.datetime.now()
+LOG_FORMAT = '%(asctime)s - %(levelname)s - %(name)s - %(message)s'
+logger = logging.getLogger(__name__)
 
 
 @plac.annotations(
-    wd_json=("Path to the downloaded WikiData JSON dump.", "positional", None, Path),
-    wp_xml=("Path to the downloaded Wikipedia XML dump.", "positional", None, Path),
+    wikidata_json=("Path to the downloaded WikiData JSON dump.", "positional", None, Path),
+    wikipedia_xml=("Path to the downloaded Wikipedia XML dump.", "positional", None, Path),
     output_dir=("Output directory", "positional", None, Path),
     model=("Model name, should include pretrained vectors.", "positional", None, str),
+    model_is_path=("Model is a path not a model name.", "flag", "mp"),
     max_per_alias=("Max. # entities per alias (default 10)", "option", "a", int),
```

Comment:
cf other comment - I think this should have worked if you passed in a path to a model - perhaps we only need to update the docstring for the `model` parameter?